src/toil/common.py

```diff
@@ -348,6 +355,15 @@ def _addOptionFn(*name, **kwargs):
                      'that do not specify an explicit value for this requirement. Standard '
                      'suffixes like K, Ki, M, Mi, G or Gi are supported. Default is %s' %
                      bytes2human( config.defaultDisk, symbols='iec' ))
+    addOptionFn("--disableSharedCache", dest="disableSharedCache", action='store_true', default=None,
+                help='Should the shared cache strategy be disabled? Default is False')
+    addOptionFn("--readGlobalFileMutableByDefault", dest="readGlobalFileMutableByDefault",
+                action='store_true', default=None, help='Shared caching disallows modification of '
```

Comment:
help should start on a new line so you don't have to wrap the string quite so narrowly.